languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -212,3 +212,23 @@ EF	EF	NCCS000
 BT	BT	AQ0CN0
 Linux	Linux	AQ0CN0
 
+# Added by Marco A.G.Pinto. Pedro told me to add here until he fixes the dictionary building.
+autoinflingir	autoinflingir	VMN0000
+Balikó	Balikó	NPMS000
+Balikó	Balikó	NPMSS00
+baratinho	baratinho	NCMS000
```

Comment:
Can we add the attribute `degree` to the diminutives (`D`) here? (see [tagset](https://github.com/languagetool-org/languagetool/blob/master/languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/tagset.txt))